.fides/systems.yml

```diff
@@ -86,3 +86,58 @@ system:
           # - fides_db.fidesuserpermissions
           # - fides_db.providedidentity
           # - fides_db.privacyrequest
+
+  # System Info
+  - fides_key: new_dictionary_system
```

Comment:
Why is it "new"?

Reply:
I misunderstood the purpose of this file! I'll just remove this.